damnit/gui/main_window.py

```diff
@@ -437,8 +437,54 @@ def _create_menu_bar(self) -> None:
         # Table menu
         action_columns = QtWidgets.QAction("Select && reorder columns", self)
         action_columns.triggered.connect(self.open_column_dialog)
+        self.autoscroll_checkbox = QtWidgets.QCheckBox('Autoscroll', self)
+        self.autoscroll_checkbox.setStyleSheet("QCheckBox"
+                               "{"
+                               "padding-left : 10px;"
+                               "padding-bottom: 7px"
+                               "}")
+        self.autoscroll_checkbox.setChecked(True)        
         tableMenu = menu_bar.addMenu("Table")
+        action_autoscroll =  QtWidgets.QWidgetAction(menu_bar)
+        action_autoscroll.setDefaultWidget(self.autoscroll_checkbox)
```

Comment:
I don't think you need to fiddle around with a checkbox and a stylesheet - Qt already has a concept of a 'checkable' action. Make a regular QAction, then call `action_autoscroll.setCheckable(True)` and Qt will show it with a toggle-able checkmark.

https://doc.qt.io/qt-5/qaction.html#checkable-prop

Also, I wonder if 'autoscroll' is clear to users? Maybe something like 'Scroll to newly added runs'? :thinking:

Reply:
I agree with the label of the widged and changed to your suggestion. Yet, the checkable prop would not provide a very nice user experience, for upon clicking the action the context menu closes with this approach. I'd rather keep the QCheckBox for this reason. 

An example of this issue can be found [here](https://stackoverflow.com/questions/2050462/prevent-a-qmenu-from-closing-when-one-of-its-qaction-is-triggered).

Reply:
I'd prefer the menu to close when I check the box, because that's how menus usually work. I've just checked in Firefox, Pycharm, Libreoffice and Inkscape - they all have checkboxes in menus, and in all of them, clicking the checkbox closes the menu.